pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -849,12 +867,12 @@ func TestRegistriesValidation(t *testing.T) {
 	for _, test := range successTests {
 		imgcfg := newImageConfig(test.name, test.config)
 		cvcfg := newClusterVersionConfig("version", "blah.io/myuser/myimage:test")
-		blocked, err := getValidBlockedRegistries(cvcfg.Status.Desired.Image, &imgcfg.Spec)
+		blocked, err := getValidBlockedRegistries(cvcfg.Status.Desired.Image, &imgcfg.Spec, test.icspRules)
 		if err != nil {
 			t.Errorf("%s: failed", test.name)
 		}
 		for _, reg := range blocked {
-			if reg == "blah.io" {
+			if reg == "blah.io" && len(test.icspRules) == 0 {
```

Comment:
This is a non-obvious dependency between the test contents and mechanism; a new `expectedBlocked` member of `successTetss` would be cleaner.

Reply:
fixed

Reply:
`expectedBlocked` is never set now?!

I was thinking
```go
expectedBlocked []string
…
assert.Equal(t, test.expectedBlocked, blocked)
```
and the `"blah.io"` constant is moved from the test code into the data.

Reply:
(Still outstanding.)